website/src/components/Install/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react'
+
+import styles from './install.module.scss'
+
+const installText = `
```

Comment:
Can we pass this as a prop, in case there are instances where we want other code to be used?

Reply:
We could, but since this is a feature parity improvement I would rather implement new features as they arise and when they are needed. So I'm leaving this for now.